### DISCUSSION TITLE
CSV import limit

### DISCUSSION THREAD

**Discussion by beecho01** [ghostfolio/ghostfolio#1673] [General]
Just wondering if the limit of 20 activities via csv is an account-wide limitation or is this something that is limited specifically for free accounts, but available if upgraded or via docker?

**Comment by dtslvr** [comment]
Hi @beecho01

Exactly, the import limitation only exists with [Ghostfolio Basic](https://ghostfol.io/en/pricing).

**Reply by beecho01** [reply]
Thanks for the quick reply and letting me know!
